File: Source/Composestar/Repository/LanguageModel/ConditionExpressions/False.java
package Composestar.Repository.LanguageModel.ConditionExpressions;

import Composestar.Repository.LanguageModel.ConditionExpressions.Visitor.*;

/**
 * Summary description for False.
 */
public class False extends ConditionExpression implements IVisitable 
{
	public False()
	{
		//
		// TODO: Add Constructor Logic here
		//
	}

	public void Accept(IVisitor visitor)
	{
		visitor.VisitFalse(this);
	}
    
    public String toString(){
        return "false";
    }
}

File: Source/DotNETLanguageModel/WeaveSpec/ConditionExpressions/False.cs
#region Using directives
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml;
using System.Xml.Serialization;
using Composestar.StarLight.WeaveSpec.ConditionExpressions.Visitor;
#endregion

namespace Composestar.StarLight.WeaveSpec.ConditionExpressions
{
    /// <summary>
    /// A false literal.
    /// </summary>
    [Serializable]
    [XmlRoot("False", Namespace = "Entities.TYM.DotNET.Composestar")]
    public class False : ConditionExpression, IVisitable
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="T:False"/> class.
        /// </summary>
        public False()
        {

        } // False()

        /// <summary>
        /// Accepts the specified visitor.
        /// </summary>
        /// <param name="visitor">The visitor.</param>
        public void Accept(IVisitor visitor)
        {
            visitor.VisitFalse(this);
        } // Accept(visitor)

    } // class False
}
